lib/dal/tests/integration_test/mod.rs

```diff
@@ -22,5 +22,5 @@ mod frame;
 mod prop;
 mod property_editor;
 mod rebaser;
-mod sdf_mock;
+mod schema_variant_views;
```

Comment:
This module and naming likely don't live here long term. Just wanted a place to place the test for now.